File: homework/Double_Precision.m
x = double(1);
y = double(2);
z = double(1);
time = 0;

while (x+z) ~= double(1)
    z = double(z/y);
    x = double(1) +z;
    time = time +1
    if time == 100000
        break
    end
end

File: double_precision.cpp
#include <iostream>

using namespace std;

void double_precision(){
    double x = 1;
    double y = 1;
    double z = x +1;
    int time = 0;
    
    while (z != 1){
        x = x/2.0;
        z =  y + x;
        time = time +1.0;
        if ((z - y) == 0.0){
            break;
        }
    }
    cout << x << endl << time << endl;
}